tests/unit/credit/test_sticky_router.py

```
@@ -859,6 +859,106 @@ async def test_reassigns_to_new_worker_if_sticky_worker_gone(
         assert router._sticky_sessions["session-X"] == "worker-2"
 
 
+def _make_no_request_credit(
+    id: int = 1, corr_id: str = "vc-corr", num_turns: int = 1
```

Ruff (0.15.21)

[error] Function argument `id` is shadowing a Python builtin

(A002)